src/shared/containers/error/index.js

```diff
@@ -1,13 +1,44 @@
 import React, { Component } from 'react';
+import Container from '../../components/container';
+import Root from '../root';
+import typography from '../../components/typography/style.css';
+import image from './missing_badger.png';
+import styles from './style.css';
 
 export default class ErrorPage extends Component {
   static propTypes = {
-    children: React.PropTypes.node
+    status: React.PropTypes.number
   };
 
   render () {
+    let text;
+
+    if (this.props.status === 500) {
+      text = (
+        <div>
+          <p>{"Our server seems to be having a little strop."}</p>
+          <p>{"Please try reloading the page or come back later."}</p>
+        </div>
+      );
+    } else {
+      text = (
+        <div>
+          <p>{"The page you’re looking for seems to have gone walkies."}</p>
+          <p>{"Please use the navigation above to hunt it down."}</p>
+        </div>
+      );
+    }
```

Comment:
I wonder if it's enough to just check for a `500` status code and treat everything else as a `404`. I think it's fine for now, then we'll see if we pick anything up in testing.